modules/upgrade/pages/server-minor-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,9 @@
+[server-minor-upgrade]
+= Server - Minor Upgrade
+
+Several times the year, the {productname} team releases minor upgrades of the {productname} Server.
+These updates offer improved or completely new features.
+For more information, see the Release Notes.
+
+Installing such minor upgrades is similar to the server maintenance updates.
```

Comment:
As discussed above, maybe you should consider merging the sections.

Why am I saying this? Because if I produce an Uyuni release and at the same time they try to run an openSUSE update, they will get both things at the same time.

Reply:
ok, merged :)